From: =?utf-8?q?Jonas_=C3=85dahl?= <jadahl@gmail.com>
Date: Mon, 17 Jun 2019 18:18:12 +0200
Subject: renderer/native: Use g_set_error() instead of _cogl_set_error()

It's even a GError, so lets use the proper API.

(cherry picked from commit 1efb32d3000ca06ee3cfcc146dc812866d243619)

Bug: https://gitlab.gnome.org/GNOME/mutter/merge_requests/622
Applied-upstream: 3.32.3, commit:cded69da6137bd1fbf03cb907078ef7959290845
---
 src/backends/native/meta-renderer-native.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backends/native/meta-renderer-native.c b/src/backends/native/meta-renderer-native.c
index b3976d4..b8e0f0c 100644
--- a/src/backends/native/meta-renderer-native.c
+++ b/src/backends/native/meta-renderer-native.c
@@ -1097,7 +1097,7 @@ meta_renderer_native_egl_context_created (CoglDisplay *cogl_display,
                                       cogl_display_egl->dummy_surface,
                                       cogl_display_egl->egl_context))
     {
-      _cogl_set_error (error, COGL_WINSYS_ERROR,
+      g_set_error (error, COGL_WINSYS_ERROR,
                    COGL_WINSYS_ERROR_CREATE_CONTEXT,
                    "Failed to make context current");
       return FALSE;
